[2021-09-27 16:04:27,981] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-27T15:30:00+00:00 [queued]>
[2021-09-27 16:04:28,055] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-27T15:30:00+00:00 [queued]>
[2021-09-27 16:04:28,058] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 16:04:28,060] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 16:04:28,061] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 16:04:28,087] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): say_hello> on 2021-09-27T15:30:00+00:00
[2021-09-27 16:04:28,093] {standard_task_runner.py:52} INFO - Started process 24100 to run task
[2021-09-27 16:04:28,122] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'say_hello', '2021-09-27T15:30:00+00:00', '--job-id', '75', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpm9fl864w', '--error-file', '/tmp/tmp87mnm9ch']
[2021-09-27 16:04:28,143] {standard_task_runner.py:77} INFO - Job 75: Subtask say_hello
[2021-09-27 16:04:28,346] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.say_hello 2021-09-27T15:30:00+00:00 [running]> on host 7c1d2d2abdb5
[2021-09-27 16:04:28,617] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Veerawat
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=say_hello
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T15:30:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-27T15:30:00+00:00
[2021-09-27 16:04:28,619] {my_dag.py:11} INFO - Hello, SAKSIAM
[2021-09-27 16:04:28,620] {python.py:151} INFO - Done. Returned value was: None
[2021-09-27 16:04:28,648] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=say_hello, execution_date=20210927T153000, start_date=20210927T160427, end_date=20210927T160428
[2021-09-27 16:04:28,742] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 16:04:28,822] {local_task_job.py:261} INFO - 1 downstream tasks scheduled from follow-on schedule check
